### PR TITLE
Added command block syntax definition

### PR DESCRIPTION
**Pull request by ghost**
This will not highlight anything contained within the command block curly braces.

### CODE REVIEW

File: WDL.YAML-tmLanguage [hunk 1]
```diff
@@ -75,7 +77,7 @@ repository:
   keywords:
     patterns:
     - name: keyword.other.wdl
-      match: \s*(call|command|output|runtime|task|workflow|if|then|else|import|as|input|output|meta|parameter_meta|scatter)\s+
```
Comment (cjllanwarne) [inline WDL.YAML-tmLanguage]: I see `command` defined below, but is it also necessary to remove `output` here?

File: WDL.YAML-tmLanguage [hunk 2]
```diff
@@ -106,4 +108,14 @@ repository:
       patterns:
       - include: '#constant_placeholder'
       - include: '#escaped_char'
+
+  command_block:
+    patterns:
+    - comment: command block
+      name: command.block.wdl
+      begin: (command)\s\{\n
```
Comment (cjllanwarne) [inline WDL.YAML-tmLanguage]: I guess that this should probably be `\s*`? To cover `command{` and `command     {`, etc

Comment (cjllanwarne) [inline WDL.YAML-tmLanguage]: We probably want an equivalent for `<<< ... >>>` style command blocks

File: WDL.tmLanguage [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
```
Comment (cjllanwarne) [inline WDL.tmLanguage:2]: I'm curious why this loss of `Computer` happened but not too worried. 

I assume you used the sublime `build` function to make this compiled version?

File: WDL.YAML-tmLanguage (hunk 1 above)
Comment (ghost) [reply WDL.YAML-tmLanguage]: It was in here twice

File: WDL.YAML-tmLanguage (hunk 2 above)
Comment (ghost) [reply WDL.YAML-tmLanguage]: I wasn't sure if command{ was proper syntax, I'll fix.

File: WDL.tmLanguage (hunk 3 above)
Comment (ghost) [reply WDL.tmLanguage:2]: I did use sublime's build function.

File: WDL.YAML-tmLanguage (hunk 1 above)
Comment (cjllanwarne) [reply WDL.YAML-tmLanguage]: 🤦‍♂️

File: WDL.tmLanguage (hunk 3 above)
Comment (cjllanwarne) [reply WDL.tmLanguage:2]: Cool. So I'll just ignore this `.tmLanguage` entirely. I might even look up how to "mark file as generated" in git 🤷‍♂️

**Comment by cjllanwarne** [comment]
Looks good to me since we now have a new ticket for the `<<< ... >>>` style blocks. 👍 

Don't forget to rebase and squash to a single commit when you merge.

[Approved with PullApprove](https://pullapprove.com/broadinstitute/wdl-sublime-syntax-highlighter/pull-request/8/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/wdl-sublime-syntax-highlighter)

**Pull request merged**